docs/user-guide/getting-started.md

```diff
@@ -1,185 +1,197 @@
+```
 
-    ```bash
-    kubectl get -n ambassador service ambassador -o 'go-template={{range .status.loadBalancer.ingress}}{{print .ip "\n"}}{{end}}'
-    ```
+4. Once you enter your email address, your terminal will print something similar
+   to the following:
 
-    Your load balancer may take several minutes to provision your IP address. Repeat the provided command until you get an IP address.
+```shell
+ -> Obtaining a TLS certificate from Let’s Encrypt.
 
-    Note: If you are a **Minikube user**, Minikube does not natively support load balancers. Instead, use `minikube service list`. You should see something similar to the following:
+ Congratulations, you’ve successfully installed the Ambassador Edge Stack in your Kubernetes cluster. Visit https://random-word-3421.edgestack.me to access your Edge Stack installation and for additional configuration.
+ ```
 
-    ```bash
-    (⎈ |minikube:ambassador)$ minikube service list
-    |-------------|------------------|--------------------------------|
-    |  NAMESPACE  |       NAME       |              URL               |
-    |-------------|------------------|--------------------------------|
-    | ambassador  | ambassador       | http://192.168.64.2:31230      |
-    |             |                  | http://192.168.64.2:31042      |
-    | ambassador  | ambassador-admin | No node port                   |
-    | ambassador  | ambassador-redis | No node port                   |
-    | default     | kubernetes       | No node port                   |
-    | kube-system | kube-dns         | No node port                   |
-    |-------------|------------------|--------------------------------|
-    ```
+Your new [Edge Policy Console](/about/edge-policy-console) will open automatically in your browser at the provided URL.
 
-    Use any of the URLs listed next to `ambassador` to access the Ambassador Edge Stack.
+### Minikube Users
 
-3. Navigate to `http://<your-IP-address>` and click through the certificate warning for access to the Edge Policy Console interface. The certificate warning appears because, by default, the Ambassador Edge Stack uses a self-signed certificate for HTTPS.
-    * Chrome users should click **Advanced > Proceed to website**.
-    * Safari users should click **Show details > visit the website** and provide your password.
+If you're a Minikube user, Edge Control will not be able to provide a domain name. However, you will still be given an IP address to access the Edge Policy Console.
 
-4. To login to the [Edge Policy Console](../../about/edge-policy-console), download and install `edgectl`, the command line tool Edge Control, by following the provided instructions on the page. The Console lists the correct command to run and provides download links for the `edgectl` binary.
+Edge Control will automatically configure TLS **if your cluster is publicly accessible.**
```

Comment:
Is this intended for the Minikube section? Minikube clusters cannot be made accessible to the Internet (or if they can, that's not how they're intended to be used).

Reply:
Thats what was in the 2ksecond google doc! : )